frontend/src/bundles/common/components/tabs/styles.module.scss

```diff
@@ -36,3 +38,13 @@
     background-color: var(--color-white-100);
     border-radius: var(--b-2);
 }
+
+@media (max-width: $md) {
+    .tabs {
+        z-index: 99;
+        justify-content: space-around;
+        width: 100vw;
```

Comment:
Can we use there 100%? To make it more responsive

Reply:
menu have to take all width of the screen always, it doesn't do that when I use %

Reply:
ok)